examples/2_simple_tooling/configure.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+"""
+In this example we write a CLI tool with brigade and click to deploy configuration.
+"""
+from brigade.core import Brigade
+from brigade.core.configuration import Config
+from brigade.plugins.inventory.simple import SimpleInventory
+from brigade.plugins.tasks import data, networking, text
```

Comment:
I don't think it makes sense to have four CLI examples. I think one is probably enough.

Reply:
I wanted to have a few use cases and also I wanted to have the bits necessary to build the more elaborated workflow under `3_elaborated`. So the idea of the examples was to show the evolution from a "playbook" to a "simple tool" to a more elaborated "workflow".

Reply:
Fair enough...I find it more useful to have relatively stand-alone examples that can be used as building blocks.

Reply:
I don't think having several examples are bad. It would be nice to be able to test them though.